src/manifolds/GeneralizedGrassmann.jl

```diff
@@ -57,6 +57,28 @@ function GeneralizedGrassmann(
     return GeneralizedGrassmann{n,k,field,typeof(B)}(B)
 end
 
+@doc raw"""
+    change_metric(M::GeneralizedGrassmann, ::EuclideanMetric, p X)
+
+Change `X` to the corresponding representer of the gradient with respect to the scaled metric
```

Comment:
I think it should either be gradient or representer of the differential.

Reply:
I am not yet sure how to generally phrase is, since the generic way would be representer of a linear functional.